pkg/machinepool/machinepool.go

```diff
@@ -1519,8 +1531,7 @@ func editMachinePool(cmd *cobra.Command, machinePoolId string,
 		return fmt.Errorf("Failed to get autoscaling or replicas: '%s'", err)
 	}
 
-	if !autoscaling && replicas < 0 ||
-		(autoscaling && isMinReplicasSet && minReplicas < 0) {
+	if !autoscaling && replicas < 0 {
```

Comment:
I guess this has already been checked before in when invoking validateaEditInput()?
why do we need to check this again?

Reply:
ill check when im home but i believe interactive needs it here

Reply:
moved all to validation afte rereading when i got home

Reply:
@hunterkepley 
can you check again on this?
I am still confused why we check "if !autoscaling && replicas < 0" here. Before this, we invoke getMachinePoolReplicas() and inside it func validateEditInput() is invoked. There we already checked this condition.
```
	if !autoscaling && replicas <= 0 {
		return fmt.Errorf("Replicas must be a positive number")
	}
```
so IMO, we should remove this check here. Or maybe I miss anything in the whole flow?